modules/configuring-default-seccomp-profile.adoc

```diff
@@ -1,15 +1,75 @@
 [id="configuring-default-seccomp-profile_{context}"]
-= Configuring the default seccomp profile
-OpenShift ships with a default seccomp profile that is referenced as `runtime/default`. You can enable the default seccomp profile for a pod or container workload by setting `RuntimeDefault` as following:
+= Enabling the default seccomp profile for all pods
 
-.Example
+{product-title} ships with a default seccomp profile that is referenced as `runtime/default`. You can enable the default seccomp profile for a pod or container workload by creating a custom Security Context Constraint (SCC), editing this and adding the default seccomp profile to this custom SCC. Finally add the custom SCC to the workload's service account.
```

Comment:
The second sentence ends with a serial list that should be broken out as steps. The last sentence is part of those steps, too.

OK, I see now there are more detailed steps below. Consider deleting these last couple of sentence. That will avoid confusion, duplication, and give a smoother entry into the admonition.